src/test/java/com/hocs/test/pages/complaints/ComplaintsTriage.java

```diff
@@ -127,6 +127,11 @@ public void selectTransferToIEDET() {
 
 
     public void enterDetailsOnTriageCaptureReasonPage() {
+        if(sessionVariableCalled("caseType").equals("COMP") && sessionVariableCalled("complaintType").equals("Service") ||
+                sessionVariableCalled("caseType").equals("COMP") && sessionVariableCalled("complaintType").equals("Minor Misconduct")){
+            recordCaseData.selectRandomOptionFromDropdownWithHeading("Directorate");
+        }
+
```

Comment:
we have basepage methods for caseType checking. You can use 'compCase()' to check the session variable is COMP.

Reply:
replaced with compCase()